### PR TITLE
completed css transitions.

### PR DESCRIPTION
**Pull request by david-s-song**
https://user-images.githubusercontent.com/90665895/158718238-87cd0325-7233-41f3-af8f-ccaf7e35b583.mov

### CODE REVIEW

File: css-transitions/styles.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228 228 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transition: scale(1);
```
Comment (BrettAlbright) [inline css-transitions/styles.css]: you want to make sure to include a time for the transition

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Comment by david-s-song** [comment]
https://user-images.githubusercontent.com/90665895/158869914-d10ab140-5cce-4794-9bf4-1f9b3ecaef28.mov

File: css-transitions/styles.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228 228 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transition: scale(1) 0.5s;
```
Comment (BrettAlbright) [inline css-transitions/styles.css]: you don't want `scale(1)` here as it is a value for the transform property, just the time. Then you can add `transform: scale(1)` on another line in the .card selector

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
1 final update!

File: css-transitions/styles.css (hunk 2 above)
Comment (david-s-song) [reply css-transitions/styles.css]: I see. Got it!

**Comment by david-s-song** [comment]
https://user-images.githubusercontent.com/90665895/158897338-48c2e532-7f31-44ae-b001-9834a9d960a8.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**